tactics/eauto.ml

```diff
@@ -172,106 +161,89 @@ and prev_search_state = (* for info eauto *)
   | Init
   | State of search_state
 
+(*s Tactics handling a list of goals. *)
+
+(* first_goal : goal list sigma -> goal sigma *)
```

Comment:
What's this comment about?

Reply:
I think that this is a remnant from an even older state of the code.